### PR TITLE
Fix more unittests for MacOS and add dependencies for unittest

### PR DESCRIPTION
**Pull request by stweil**

### CODE REVIEW

**Comment by amitdo** [comment]
Fonts:

* Will this download the fonts every time the tests are invoked?
* Maybe we should have the open source fonts in one of our repos?

File: unittest/Makefile.am [hunk 1]
```diff
@@ -495,8 +498,31 @@ fonts += Times_New_Roman.ttf
 fonts += UnBatang.ttf
 fonts += Verdana.ttf
 
+# These tests depend on installed model files and fonts:
+#
+# apiexample_test baseapi_test lang_model_test layout_test
+# ligature_table_test loadlang_test lstm_recode_test lstm_squashed_test
+# lstm_test lstmtrainer_test mastertrainer_test osd_test
+# pagesegmode_test pango_font_info_test progress_test#
```
Comment (amitdo) [inline unittest/Makefile.am:506]: An unwanted '#' at EOL.

**Comment by stweil** [comment]
>     * Will this download the fonts every time the tests are invoked?

No, it should only download it once. It also respects existing symbolic links to fonts which are installed from the Linux distribution.

>     * Maybe we should have the open source fonts in one of our repos?

That is not possible for the Microsoft fonts.

File: unittest/Makefile.am (hunk 1 above)
Comment (stweil) [reply unittest/Makefile.am:506]: Ah, thanks, I'll fix that.

**Comment by amitdo** [comment]
The macOS build failed. Why?

**Comment by stweil** [comment]
> The macOS build failed. Why?

It failed when compiling `gunixconnection.c`, so this is unrelated to Tesseract code.
I think @egorpugin will have a look as soon as his time permits.

**Comment by egorpugin** [comment]
Yes, sorry for that.
I've updated pango package, it required glib.gio package what was not tested on macos yet.

**Pull request merged**